src/ai/backend/client/cli/vfolder.py

```diff
@@ -711,24 +710,21 @@ def clone(name, target_name, target_host, usage_mode, permission):
             print_error(e)
             sys.exit(ExitCode.FAILURE)
 
+    # NOTE: Tracking the progress from the storage-proxy is not supported yet. (See #1033)
     async def clone_vfolder_tracker(bgtask_id):
-        print_wait(
-            "Cloning the vfolder... "
-            "(This may take a while depending on its size and number of files!)",
-        )
         async with AsyncSession() as session:
             try:
                 bgtask = session.BackgroundTask(bgtask_id)
                 completion_msg_func = lambda: print_done("Cloning the vfolder is complete.")
                 async with bgtask.listen_events() as response:
-                    # TODO: get the unit of progress from response
-                    with tqdm(unit="bytes", disable=True) as pbar:
+                    async with Spinner(
+                        "Cloning the vfolder... "
+                        "(This may take a while depending on its size and number of files!)",
+                    ):
```

Comment:
```
async with (
  bgtask.listen_events() as response,
  Spinner(
    "Cloning the vfolder... "
    "(This may take a while depending on its size and number of files!)",
  ):
  ...
```
we can use [parenthesized context manager](https://docs.python.org/3.10/whatsnew/3.10.html#parenthesized-context-managers)

Reply:
Oh that's good!

Reply:
The discussion is ongoing about how to use the spinner and progress, so I'll leave it for now and fix it after the discussion.